pkg/ccl/logictestccl/testdata/logic_test/zone

```diff
@@ -309,6 +309,47 @@ SHOW ZONE CONFIGURATION FOR PARTITION x1 OF TABLE t35756
           constraints = '[]',
           lease_preferences = '[]'
 
+statement ok
+CREATE TABLE infect (x INT PRIMARY KEY);
+ALTER TABLE infect PARTITION BY LIST (x) ( PARTITION p1 VALUES IN (1));
+ALTER INDEX infect@primary CONFIGURE ZONE USING num_replicas=5;
+ALTER PARTITION p1 OF TABLE infect CONFIGURE ZONE USING constraints='[+dc=dc1]'
+
+query TT
+SELECT partition_name, zone_config FROM [SHOW PARTITIONS FROM TABLE infect]
+----
+p1 constraints = '[+dc=dc1]'
```

Comment:
I think the above lines are from a different PR, but presumably you'll see the conflict when you rebase.